web3-utils/config.ts

```diff
@@ -0,0 +1,4 @@
+export const privateKey = "96056e9ee8751520722534bf43f18026447dce015ad0217cf182e85d0d81589d"
+export const address = "0xecfe4b91b3f094496610fb0a947f5b44b212173f"
+export const url = "http://geth:8545"
```

Comment:
Address here should be provided. I'm not sure that hardcoding is the way. Also, not sure if inside of GH actions we will be able to do a lot of stuff with `http`

Reply:
I don't follow the concern about `http`.  http is used to talk between the js test and the geth instance via the docker network.